encoding.bs

```diff
@@ -612,6 +612,17 @@ prescribes, as that is necessary to be compatible with deployed content.
 <a>labels</a> are also available as non-normative
 <a href=encodings.json>encodings.json</a> resource.
 
+<p class=note>The set of supported <a for=/>encodings</a> is primarily based on the intersection
+of the sets supported by major browser engines when the development of this standard started and
+removing encodings that were rarely used legitimately but that could be used in attacks.
```

Comment:
I found this sentence a little hard to parse. How about

> The set of supported <a for=/>encodings</a> is primarily based on the intersection
> of the sets supported by major browser engines when the development of this standard started, **_while_**
> removing encodings that were rarely used legitimately but that could be used in attacks.

Reply:
Thanks. Fixed.